scripts/common.py

```diff
@@ -38,9 +39,34 @@
 NODE_MODULES_PATH = os.path.join(CURR_DIR, 'node_modules')
 FRONTEND_DIR = os.path.join(CURR_DIR, 'core', 'templates', 'dev', 'head')
 YARN_PATH = os.path.join(OPPIA_TOOLS_DIR, 'yarn-v1.17.3')
+OS_NAME = platform.system()
+ARCHITECTURE = platform.machine()
+
 # Add path for node which is required by the node_modules.
 os.environ['PATH'] = (
-    '%s/bin:' % NODE_PATH + '%s/bin:' % YARN_PATH + os.environ['PATH'])
+    '%s%sbin%s' % (NODE_PATH, os.sep, os.pathsep) +
```

Comment:
You could use os.path.join to simplify some of this stuff, I think.

Reply:
Updated